mistletoe/block_token.py

```diff
@@ -958,7 +958,16 @@ class HTMLBlock(BlockToken):
                                 span_token._closing_tag)) + r')\s*$')
 
     def __init__(self, lines):
-        self.content = ''.join(lines).rstrip('\n')
+        self.children = (span_token.RawText(''.join(lines).rstrip('\n')),)
+
+    @property
+    def content(self):
+        """Return the raw HTML content.
+
+        Convenience wrapper to preserve backward compatibility.
+        New code should use the `children` attribute instead.
```

Comment:
Hmm, now that I think of it, the client code looks better when you use the property getter. Maybe we shouldn't discourage its use, but instead add similar property getters on the other leaf block tokens, too?

Reply:
Well, good question. I think we could safely put the `content` property getter also into `BlockCode` and `CodeFence` as suggested, because besides `HTMLBlock` these are the only leaf block tokens which should always have just one `RawText` child, right? Of course, we should be careful when introducing new API like this - I can't see no gotchas here though, can you?

Reply:
Yes, I think it should be perfectly safe, and make for a better API. I'll go ahead and add those.

Reply:
Done, OK.